Primo.js

```diff
@@ -442,6 +447,7 @@ function importPNX(text) {
 			item.attachments.push({url: ZU.xpathText(doc, '//display/lds03'), title: "HOLLIS Permalink", snapshot: false});		
```

Comment:
Yeah, the extra call might not be worth for this information. Maybe we can also construct the permalink from the id and url... Well, as you mentioned, maybe for another time.